src/plugins/vis_type_drilldown/public/drilldown_options.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback, useState } from 'react';
+import {
+  EuiPanel,
+  EuiTitle,
+  EuiTextArea,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFieldText,
+  EuiAccordion,
+} from '@elastic/eui';
+import { FormattedMessage } from '@osd/i18n/react';
+
+import { VisOptionsProps } from 'src/plugins/vis_default_editor/public';
+import { DrilldownVisParams } from './types';
+
+function DrilldownOptions({ stateParams, setValue }: VisOptionsProps<DrilldownVisParams>) {
+  const onMarkdownUpdate = useCallback(
```

Comment:
nit - missed a rename here 😄  - maybe one more search for `markdown` strings in this plugin.